replicator/src/main/scala/com/evolutiongaming/kafka/journal/replicator/commit/AsyncPeriodicTopicCommit.scala

```diff
@@ -96,6 +97,17 @@ private[replicator] final class AsyncPeriodicTopicCommit[F[_]: MonadThrowable] p
       ).as(true)
     }
   }
+
+  /**
+   * `kafka.journal.KafkaConsumer` wraps exceptions in `KafkaConsumerError` - to be on the safe
+   * side, this method allows handling Java consumer errors both wrapped and unwrapped.
+   */
+  private def matchConsumerError[T](pf: PartialFunction[Throwable, T]): PartialFunction[Throwable, T] = {
+    pf.orElse {
+      case e: com.evolutiongaming.kafka.journal.KafkaConsumerError if pf.isDefinedAt(e.cause) =>
```

Comment:
why fully qualified name?

Reply:
To be sure it's the one coming from KJ, not some random Java consumer exception